src/components/directory-content.js

```diff
@@ -817,6 +818,19 @@ const DirectoryContent = () => {
         );
     }
 
+    const emptyMenu = () => {
+        return !(
+            allowsRename() ||
+            allowsPublishability() ||
+            allowsExport() ||
+            allowsDelete() ||
+            allowsCopyContingencyToScript() ||
+            allowsReplaceContingencyWithScript() ||
+            allowsCopyFilterToScript() ||
+            allowsReplaceFilterWithScript()
+        );
+    };
+
```

Comment:
In general, a lot a logic is implemented in thos allowX methods. Maybe a back service could give us the allowed actions ?

Reply:
It's an idea ! we could discuss of this new US with the team ! it allows to compute allowed actions only once from back service and not from the front.

Reply:
Yes, to be done late